### PR TITLE
Command description tweaks

### PR DESCRIPTION
**Pull request by lostintangent**
This PR simply makes a few small tweaks to the descriptions for some of the access-key commands.

### CODE REVIEW

**Comment by geof90** [comment]
LGTM!

File: cli/script/command-parser.ts [hunk 1]
```diff
@@ -44,9 +44,9 @@ function accessKeyAdd(commandName: string, yargs: yargs.Argv): void {
...
         .demand(/*count*/ 3, /*max*/ 3)  // Require exactly three non-option arguments.
-        .example("access-key " + commandName + " \"VSTS Integration\"", "Creates a new access key with the name \"VSTS Integration\" which expires by default in 60 days")
-        .example("access-key " + commandName + " \"One time key\" --ttl 5m", "Creates a new access key with the name \"One time key\" which expires in 5 minutes")
-        .option("ttl", { default: null, demand: false, description: "A duration string specifying the time for which the access key remains valid for use", type: "string" });
+        .example("access-key " + commandName + " \"VSTS Integration\"", "Creates a new access key with the name \"VSTS Integration\", which expires in 60 days")
+        .example("access-key " + commandName + " \"One time key\" --ttl 5m", "Creates a new access key with the name \"One time key\", which expires in 5 minutes")
+        .option("ttl", { default: "60d", demand: false, description: "Duration string which specifies the amount of time that the access key should remain valid for (e.g 5m, 60d, 1y)", type: "string" });
```
Comment (scottbommarito) [inline cli/script/command-parser.ts]: does the amount of time support specifying a length in months, like `2mo` or `3mo`? if it does, might be useful to put here and below in your `(e.g...`

Comment (scottbommarito) [inline cli/script/command-parser.ts]: also, this is totally minor, but given that we're using the not-immediately obvious abbreviation `ttl`, i would like to see some small indication of what it stands for somewhere in here, but i'm not sure the best way to do that

**Comment by scottbommarito** [comment]
`// LGTM`, just some super minor comments

**Comment by shishirx34** [comment]
LGTM

File: cli/script/command-parser.ts (hunk 1 above)
Comment (geof90) [inline cli/script/command-parser.ts]: Months have to be spelt in full (e.g. '6months'), and we already linked to how to craft this string in the docs, I don't think we have to show every single example here.

[Hmm TTL is a super commonly used term](http://lmgtfy.com/?q=ttl&l=1), I don't really think it needs spelling out in full..

Comment (lostintangent) [inline cli/script/command-parser.ts]: Yeah my intention here wasn't to outline all possible values, but rather, to provide some examples which illustrate the syntax. Also, I'm comfortable with the use of TTL here, which is why we named the parameter that in the first place.

**Pull request merged**